src/System.Net.Http.WinHttpHandler/tests/UnitTests/FakeInterop.cs

```diff
@@ -314,8 +321,9 @@ public static bool WinHttpQueryDataAvailable(SafeWinHttpHandle requestHandle, ou
             uint bufferSize,
             out uint bytesWritten)
         {
-            if (TestControl.WinHttpAPIFail)
+            if (TestControl.Fail.WinHttpWriteData)
             {
+                // TODO: Set last error ?
```

Comment:
Actually, those TODO's were left in there by accident.  I will remove them.

I don't need to set the last error in those particular methods in the Fake layer since we don't have any tests that test against the values (yet).  If needed, the Fake layer will be revised in the future as more complexity is required due to adding tests.